docs/exercises_format.md

```diff
@@ -60,9 +60,34 @@ An exercise is described by a directory containing at most the following files:
 
 Json description with the following format (in version "1"):
 ```
-{ "learnocaml_version": <string>,
-  "kind": "exercise" | "problem" | "project",
-  "stars": [1 .. 5]
+{ "learnocaml_version" : "1",
+  "kind"               : "exercise" | "problem" | "project",
+  "stars"              : [1 .. 5]
+}
+```
+
+Json description in version "2" contains more metadata:
+```
+{
+  "learnocaml_version" : "2",
+  "kind"               : "exercise" | "problem" | "project",
+  "stars"              : [1 .. 5],
+  /* In an exercise repository, each exercise must have a unique identifier. */
+  "identifier"         : "some_unique_identifier",
+  /* Authors with their emails. */
+  "authors"            : ["Xavier Leroy <some@email>", "Damien Doligez <someother@email>"],
```

Comment:
In terms of implementation, maybe the author field could be a tuple (this is simply to avoid parsing the `< .. >` as the email field, which is indeed quite simply).

Reply:
Good idea. I fix that.